internals/secrethub/migrate.go

```diff
@@ -412,80 +481,153 @@ func (cmd *MigrateApplyCommand) Run() error {
 		return fmt.Errorf("op is signed in to a different account than planned. Run `eval $(op signin %s) to login to the desired account or change the sign-in-address in the plan", plan.SignInAddress)
 	}
 
-	fmt.Fprintln(cmd.io.Output(), "validating the migration plan...")
-	for _, vault := range plan.vaults {
-		exists, err := onepassword.ExistsVault(vault.Name)
-		if err != nil {
-			return fmt.Errorf("could not check vault existence: %s", err)
-		}
-		if exists {
-			for _, item := range vault.Items {
-				exists, err = onepassword.ExistsItemInVault(vault.Name, item.Name)
-				if err != nil {
-					return fmt.Errorf("could not check item existence: %s", err)
-				}
-				if exists {
-					return fmt.Errorf("an item named %s already exists in vault %s. Remove the item from the plan if you've already migrated it before. If not, you can rename the item in the plan", item.Name, vault.Name)
-				}
-			}
-		}
-	}
-	fmt.Fprintf(cmd.io.Output(), "plan is ok\n\n")
-
 	client, err := cmd.newClient()
 	if err != nil {
 		return err
 	}
 
+	createCount := 0
+	warningCount := 0
+	updateCount := 0
+	skipCount := 0
+	alreadyUpToDateCount := 0
+
+	var changes []change
+
 	i := 1
 	for _, vault := range plan.vaults {
-		exists, err := onepassword.ExistsVault(vault.Name)
+		if i != 1 {
+			fmt.Fprintf(cmd.io.Output(), "\033[1A\033[K")
```

Comment:
That's a good point. I'm not sure if it will look okay and I don't think it's worth looking into it too much, the previous progress indication was fine imo, so I'll just stick with that.